server/graphql/v2/input/UpdateAttributesInput.js

```diff
@@ -0,0 +1,19 @@
+import { GraphQLBoolean, GraphQLInputObjectType, GraphQLString } from 'graphql';
+
+import { IsoDateString } from '../../v1/types';
+
+import { AccountReferenceInput } from './AccountReferenceInput';
+
+export const UpdateAttributesInput = new GraphQLInputObjectType({
+  name: 'UpdateAttributesInput',
+  description: 'Input type for UpdateType',
+  fields: () => ({
+    id: { type: GraphQLString },
+    slug: { type: GraphQLString },
+    title: { type: GraphQLString },
+    isPrivate: { type: GraphQLBoolean },
+    makePublicOn: { type: IsoDateString },
```

Comment:
From `server/graphql/v2/scalar/ISODateTime.ts`

```suggestion
    makePublicOn: { type: ISODateTime },
```